src/stories/components/MainMenu/MainMenu.js

```diff
@@ -22,7 +22,7 @@ export const MainMenu = () => {
   useEffect(() => {
     if (currentUser) {
       // setError('');
-      database.getProfile(currentUser.uid)
+      USERS.getUser(currentUser.uid)
         .then((doc) => {
           setProfile(doc.data());
         })
```

Comment:
catch block might not work since we are not returning a promise now. More details in the first comment